selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -101,6 +101,16 @@ void Networking::wrongPassword(const QString &ssid) {
   }
 }
 
+void Networking::showEvent(QShowEvent* event) {
+  // Wait to refresh to avoid lag when cycling through settings widgets
```

Comment:
```suggestion
  // Wait to refresh to avoid queuing up too many scans if shown and hidden quickly
```

Reply:
Scans aren't really a problem, it's the blocking calls done when refreshing the networks, it causes significant lag if you click and click out of the Network pane